go/server/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"github.com/feast-dev/feast/go/feast"
+	"github.com/feast-dev/feast/go/protos/feast/serving"
+	"google.golang.org/grpc"
+	"log"
+	"net"
+	"os"
+	"strings"
+)
+
+const (
+	flagFeastRepoPath    = "FEAST_REPO_PATH"
+	flagFeastRepoConfig  = "FEAST_REPO_CONFIG"
+	flagFeastGrpcPort    = "FEAST_GRPC_PORT"
+	defaultFeastGrpcPort = "6566"
+	feastServerVersion   = "0.18.0"
+	defaultFeastHttpPort = "8081"
+)
+
+func main() {
+
+	repoPath := os.Getenv(flagFeastRepoPath)
+	repoConfig := os.Getenv(flagFeastRepoConfig)
+
+	if repoPath == "" && repoConfig == "" {
+		log.Fatalln(fmt.Sprintf("One of %s of %s environment variables must be set", flagFeastRepoPath, flagFeastRepoConfig))
+		return
+	}
+	config, err := feast.NewRepoConfig(repoPath, repoConfig)
+	if err != nil {
+		log.Fatalln(err)
+		return
+	}
+	log.Println("Initializing feature store...")
```

Comment:
We should add a `TODO` to add a proper logging library like https://github.com/Sirupsen/logrus